src/sentry/integrations/api/endpoints/organization_coding_agents.py

```diff
@@ -82,3 +172,279 @@ def get(self, request: Request, organization) -> Response:
+                    continue
+                prompt = get_coding_agent_prompt(run_id, trigger_source)
+
+                if not prompt:
+                    logger.warning(
+                        "coding_agent.prompt_not_available",
+                        extra={
+                            "organization_id": organization.id,
+                            "run_id": run_id,
+                            "repo_name": repo_name,
+                            "trigger_source": trigger_source,
+                        },
+                    )
+                    continue
+
+                launch_request = CodingAgentLaunchRequest(
+                    prompt=prompt,
+                    repository=repo,
+                    branch_name=sanitize_branch_name(autofix_state.request["issue"]["title"]),
+                )
+
+                # Launch the agent
+                coding_agent_state = installation.launch(launch_request)
+
+                # Store the coding agent state to Seer
+                repo_store_success = store_coding_agent_state_to_seer(
+                    run_id=run_id,
+                    coding_agent_state=coding_agent_state,
+                )
+
+                if not repo_store_success:
+                    logger.warning(
+                        "coding_agent.seer_store_failed",
+                        extra={
+                            "organization_id": organization.id,
+                            "run_id": run_id,
+                            "repo_name": repo_name,
+                        },
+                    )
+
+                results.append(
+                    {
+                        "repo_name": repo_name,
+                        "coding_agent_state": coding_agent_state,
+                    }
+                )
+
+            except Exception as e:
+                logger.exception(
+                    "coding_agent.repo_launch_error",
+                    extra={
+                        "organization_id": organization.id,
+                        "run_id": run_id,
+                        "repo_name": repo_name,
+                        "error": str(e),
+                    },
+                )
+                # Continue with other repos instead of failing entirely
+                continue
+
+        return results
+
+    def post(self, request: Request, organization) -> Response:
+        """Launch a coding agent."""
+        if not features.has("organizations:seer-coding-agent-integrations", organization):
+            return Response({"detail": "Feature not available"}, status=404)
+
+        try:
+            # Validate request and get integration
+            validation_result = self._validate_and_get_integration(request, organization)
+            if isinstance(validation_result, Response):
+                return validation_result
+            integration, installation = validation_result
+
+            # Get autofix state
+            run_id = request.data.get("run_id")
```

Comment:
please use a request serializer for data validation. e.g. https://github.com/getsentry/sentry/blob/26595aa9516d6cea269e452758d03a31b54c29ba/src/sentry/api/endpoints/organization_member/team_details.py#L400